<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioContent</class>
 <widget class="QWidget" name="AudioContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conteúdo em Áudio</string>
  </property>
  <widget class="QLabel" name="lbl_begin_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tempo de Início:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_duration">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Duração (segundos):</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_duration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>14</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_end_time">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tempo de Término:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_begin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>14</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_end">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>14</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choose_audio">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Escolher Áudio</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/folder_open.png</normaloff>:/i/folder_open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_media_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QWidget" name="audio_player_widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>694</width>
     <height>92</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="volume_control_widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>301</width>
     <height>237</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/check.png</normaloff>:/i/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/no.png</normaloff>:/i/no.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
